django_project/frontend/src/app/index.jsx

```diff
@@ -13,31 +13,33 @@
  * __copyright__ = ('Copyright 2023, Unicef')
  */
 
-import React, { Component } from 'react';
-import { createRoot } from "react-dom/client";
-import { Provider } from 'react-redux';
+import React, {Component} from 'react';
+import {createRoot} from "react-dom/client";
+import {Provider} from 'react-redux';
 import c from 'classnames';
 import NavBar from '../components/Navbar';
-import { EmbedConfig } from "../utils/embed";
-import { urlParams } from "../utils/main";
+import {EmbedConfig} from "../utils/embed";
```

Comment:
hi @indraneel 
Could you change the format rule for this?
To prevent this changes

Thank you